intake_elasticsearch/elasticsearch_seq.py

```diff
@@ -77,12 +87,25 @@ def _run_query(self, size=None, end=None):
         self.es.clear_scroll(scroll_id=sid)
         return s
 
+    def to_dask(self):
+        import dask.bag as db
+        from dask import delayed
+        self.discover()
+        parts = []
+        if self.npartitions == 1:
+            parts.append(delayed(self._get_partition)(0))
+        else:
+            for slice_id in range(self.npartitions):
+                parts.append(
+                    delayed(self._get_partition)((slice_id, self.npartitions)))
+        return db.from_delayed(parts)
+
     def _get_schema(self, retry=2):
         """Get schema from first 10 hits or cached dataframe"""
         return base.Schema(datashape=None,
                            dtype=None,
                            shape=None,
-                           npartitions=1,
+                           npartitions=self.npartitions,
                            extra_metadata={})
 
     def _get_partition(self, _):
```

Comment:
Would be nice to give the variable another name than `_`, now that it actually does something (and it should appear in the docstring, saying it can be a tuple).

Reply:
Yes, to be honest, I didn't like it. You also call `_get_partition` with 0. In your `intake` documentation you mention that it's signature is `_get_partition(i)` where i is the number of partition, we need to get. Maybe something like:

```python
def _get_partition(partition=None):
    slice_id, slice_max = (partition, self.npartitions) if partition else (None, None)

    # rest of code here
```
`None` for partition would means get all
That way we keep the same signature as described in the `intake` documentation.

Reply:
Thank sounds fine

Reply:
Fixed you can check it. Have an issue with setting the tuple and using it in the `_run_query` so went with a slight different method.